addons/hr_work_entry/models/hr_work_entry.py

```diff
@@ -220,13 +219,13 @@ def _error_checking(self, start=None, stop=None, skip=False, employee_ids=False)
             start = start or min(self.mapped('date_start'), default=False)
             stop = stop or max(self.mapped('date_stop'), default=False)
             if not skip and start and stop:
-                domain = [
-                    ('date_start', '<', stop),
-                    ('date_stop', '>', start),
-                    ('state', 'not in', ('validated', 'cancelled')),
-                ]
+                domain = (
+                    Domain('date_start', '<', stop)
+                    & Domain('date_stop', '>', start)
+                    & Domain('state', 'not in', ('validated', 'cancelled'))
+                )
```

Comment:
Is this different from 
```suggestion
                domain = Domain([('date_start', '<', stop), ('date_stop', '>', start), ('state', 'not in', ('validated', 'cancelled'))])
```
?

Reply:
It's exactly the same. Calling `Domain` with a list will build that same domain, it will just instantiate and parse that list to arrive to the same result.